XamarinCommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -67,6 +67,12 @@
     <Compile Include="**/*.shared.*.cs" />
     <None Include="..\LICENSE" PackagePath="" Pack="true" />
     <None Include="..\assets\XamarinCommunityToolkit_128x128.png" PackagePath="icon.png" Pack="true" />
+    <Compile Include="Helpers\BindablePropertyParameters.cs">
+      <SubType></SubType>
+    </Compile>
+    <Compile Include="Helpers\BindablePropertyBuilder.cs">
+      <SubType></SubType>
+    </Compile>
```

Comment:
Please fix file names and discard this file (This file should be preserved non-changed)

File names should be ```BindablePropertyParameters.shared.cs``` and ```BindablePropertyBuilder.shared.cs```